src/server/scripts/Battlegrounds/SeethingShore/battleground_seething_shore.cpp

```diff
@@ -0,0 +1,902 @@
+                    SeethingShore::Spells::AchievementTrackerCreditClaimJumper,
+                    SeethingShore::Spells::AchievementTrackerDeathFromAbove
+                });
+    }
+
+    void ApplyParachuteAura(SpellEffIndex /*effIndex*/) const
+    {
+        if (Player* player = GetHitPlayer())
+        {
+            CastSpellExtraArgs args;
+            args.TriggerFlags = TRIGGERED_FULL_MASK;
+            args.OriginalCastId = GetSpell()->m_castId;
+
+            player->CastSpell(player, static_cast<uint32>(GetEffectInfo().CalcValue(player)), args);
+            player->CastSpell(player, SeethingShore::Spells::AchievementTrackerCreditClaimJumper, true);
+            player->CastSpell(player, SeethingShore::Spells::AchievementTrackerDeathFromAbove, true);
+        }
+    }
+
+    void ResetFallDamage(SpellEffIndex /*effIndex*/) const
+    {
+        if (Player* player = GetHitPlayer())
+            player->SetFallInformation(0, player->GetPositionZ());
+    }
+
+    void Register() override
+    {
+        OnEffectHitTarget += SpellEffectFn(spell_bg_seething_shore_parachute::ApplyParachuteAura, EFFECT_0, SPELL_EFFECT_APPLY_AURA);
+        OnEffectHitTarget += SpellEffectFn(spell_bg_seething_shore_parachute::ResetFallDamage, EFFECT_1, SPELL_EFFECT_KNOCK_BACK);
+    }
+};
+
+// 294701 - Speed Up
+class spell_bg_seething_shore_speed_up : public SpellScript
+{
+    void ResetCooldowns(SpellEffIndex /*effIndex*/) const
+    {
+        // Unsure about which spells should reset and which not
+        // It's not the same list as the ones that reset on encounter reset
+        if (Player* player = GetCaster()->ToPlayer())
+            player->GetSpellHistory()->ResetAllCooldowns();
+    }
+
+    void Register() override
+    {
+        OnEffectLaunch += SpellEffectFn(spell_bg_seething_shore_speed_up::ResetCooldowns, EFFECT_1, SPELL_EFFECT_SCRIPT_EFFECT);
+    }
+};
+
+struct go_bg_seething_shore_azerite : GameObjectAI
+{
+    explicit go_bg_seething_shore_azerite(GameObject* go) : GameObjectAI(go) { }
+
+    bool OnCapturePointAssaulted(Player* player) override
+    {
+        if (me->GetGoState() != GO_STATE_READY || me->HasFlag(GO_FLAG_NOT_SELECTABLE))
+            return true;
+
+        if (ZoneScript* zoneScript = me->GetZoneScript())
+        {
+            zoneScript->DoAction(SeethingShore::Actions::CaptureAzeriteNode, player, me);
+            return false;
+        }
+
+        return true;
+    }
+};
+
+struct transport_seething_shore : TransportScript
+{
+    explicit transport_seething_shore() : TransportScript("transport_seething_shore") { }
+
+    void OnRemovePassenger(Transport* /*transport*/, Player* player) override
+    {
+        if (Battleground const* bg = player->GetBattleground())
+        {
+            if (bg->GetStatus() != STATUS_IN_PROGRESS)
+            {
+                WorldSafeLocsEntry const* pos = bg->GetTeamStartPosition(Battleground::GetTeamIndexByTeamId(player->GetBGTeam()));
```

Comment:
can we assume that pos is never null?

Reply:
People can't enter the bg without a valid one, that said with `.reload battleground` it could be possible to fill in bad data in the database during a running battleground and cause some issues